core/tests/protractor_utils/LibraryPage.js

```diff
@@ -81,6 +82,19 @@ var LibraryPage = function() {
     return waitFor.pageToFullyLoad();
   };
 
+  this.addToPlaylist = function(index) {
+    var addToPlaylistButton = element.all(by.css(
+      '.protractor-test-add-to-playlist-btn')
+    ).get(index);
+
+    browser.actions().mouseMove(element.all(
+      by.css('.protractor-test-exp-summary-tile-title')).get(index)).perform();
+
+    waitFor.elementToBeClickable(addToPlaylistButton,
```

Comment:
break line after `(`.

Reply:
@DubeySandeep I have made the required changes but why github is not showing these as outdated? See my last commit.

Reply:
Can you please add a link to the commit?

Reply:
Oh, It's fixed. Thanks, Done